src/windows/main_window.py

```diff
@@ -429,6 +429,9 @@ def open_project(self, file_path, clear_thumbnails=True):
         """ Open a project from a file path, and refresh the screen """
 
         app = get_app()
+        settings = app.get_settings()
+        # Set default load-project path to this path.
+        settings.setDefaultPath(settings.actionType.LOAD, os.path.dirname(file_path))
```

Comment:
Ditto on file_path being reduced to folder name here - already handled in setDefaultPath()